src/js/core/range.js

```diff
@@ -605,7 +605,25 @@ define([
        * @return {WrappedRange}
        */
       createFromNode: function (node) {
-        return this.create(node, 0, node, 1);
+        var sc = node;
+        var so = 0;
+        var ec = node;
+        var eo = dom.nodeLength(ec);
+
+        // browsers can't target a picture or void node
+        if (dom.isVoid(sc) || dom.isImg(sc)) {
+          so = dom.listPrev(sc).length - 1;
+          sc = sc.parentNode;
+        }
+        if (dom.isBR(ec)) {
+          eo = dom.listPrev(ec).length - 1;
+          ec = ec.parentNode;
+        } else if (dom.isVoid(ec) || dom.isImg(sc)) {
+          eo = dom.listPrev(ec).length;
```

Comment:
Some browsers like IE can't target directly an image but target the parentNode with the good offset.

When the browser target a br, the caret must be on the begin of the br tag (in this case when the user insert a char and the tag contains only a br the browser remove the br).

But for a void like image we want to select all the void or image and not only the start.